include/wavm/WAVMWasmModule.h

```diff
@@ -129,8 +127,6 @@ class WAVMWasmModule final
 
     uint32_t allocateThreadStack();
 
-    std::unique_ptr<openmp::PlatformThreadPool>& getOMPPool();
```

Comment:
The thread pool was good for local performance, but added complexity which we need to remove in order to redo the implementation